pkg/framework/aries/framework.go

```diff
@@ -39,7 +40,7 @@ func New(opts ...Option) (*Aries, error) {
 		return nil, errors.Errorf("default option initialization failed: %w", err)
 	}
 
-	frameworkOpts := &Aries{}
+	frameworkOpts := &Aries{newProtocolSvc: make([]api.NewProtocolSvc, 0)}
```

Comment:
redundant: initializing with size 0,
append won't cause panic when slice is nil.

Reply:
done